### ISSUE TITLE
Possible to sort group feeds chronologically? (?sorting_setting=CHRONOLOGICAL)

### ISSUE THREAD

**Issue by nadir7379** [kevinzg/facebook-scraper#935]
On the desktop web version of Facebook, you are able to sort group feeds in three different ways:

<img width="354" alt="image" src="https://user-images.githubusercontent.com/1503657/205123326-5b08349d-4112-4f40-be13-510904b4b614.png">

**The issue we face now is:** We would like to sort our group feeds by 'Newest posts', on the mobile web version however (`m.facebook.com`) this option cannot be found. Per default, feeds are sorted by 'Newest activity' (on mobile).

On the desktop web version of Facebook, adding the parameter  `?sorting_setting=CHRONOLOGICAL` does the job. However, for mobile URLs this does not work. 

**Therefore: Is there any way to sort group feeds chronologically on mobile?**

**Comment by sagihgithub** [comment]
Hi
Did you find an answer ?

**Comment by zeglin** [comment]
I'm also interested in this and trying to find a solution to sort mbasic or mobile chronologically.

**Comment by henriquetguedes** [comment]
I added `?sorting_setting=CHRONOLOGICAL` in line 45 of [page_iterators.py](https://github.com/kevinzg/facebook-scraper/blob/30a1f958cab850de6c2b06040523b4ddec26422c/facebook_scraper/page_iterators.py#L45) after the last forward-slash (the string being formatted with the group name/id) and it seems not to have broken anything. Hard to tell if the results are perfectly consistent with the Chronological sort, as I don't want to get banned for good and would need a deeper scrape to ascertain that, but no evidence saying otherwise so far.

seems to help with issues #518 and #998

**Comment by s22f32dd** [comment]
> I added `?sorting_setting=CHRONOLOGICAL` in line 45 of [page_iterators.py](https://github.com/kevinzg/facebook-scraper/blob/30a1f958cab850de6c2b06040523b4ddec26422c/facebook_scraper/page_iterators.py#L45) after the last forward-slash (the string being formatted with the group name/id) and it seems not to have broken anything. Hard to tell if the results are perfectly consistent with the Chronological sort, as I don't want to get banned for good and would need a deeper scrape to ascertain that, but no evidence saying otherwise so far.
> 
> seems to help with issues #518 and #998

did you mean this way?
`  start_url = utils.urljoin(FB_MOBILE_BASE_URL, f'groups/{group}/?sorting_setting=CHRONOLOGICAL')`

  I tried this and unfortunately this is not working for me. Maybe I am doing something wrong?

**Comment by henriquetguedes** [comment]
Precisely. I thought it worked back then, maybe not but I've had to pause the project.
